app.rb

```diff
@@ -0,0 +1,93 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+
+class App
+  attr_accessor :books, :teacher, :student, :rentals
+
+  def initialize
+    @books = []
+    @teacher = []
+    @student = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each_with_index { |book, i| print "(#{i}) Title: \"#{book.title}\", Author: \"#{book.author}\"\n" }
+  end
+
+  def list_people
+    people = [*@teacher, *@student]
+    people.each_with_index { |person, i| print "(#{i}) Name: \"#{person.name}\", Age: \"#{person.age}\"\n" }
+  end
+
+  def create_person # rubocop:disable Metrics/MethodLength
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    input = gets.chomp.to_i
+    case input
+    when 1
```

Comment:
- [ ] As mentioned by the previous reviewer and as required for the project, kindly ensure your application works as expected. As pointed out in the last review, right now users cannot create a new `Person` for both `teacher` and `student`. Below are the error images, I am getting when trying so:

| Teacher Error | Student |
| ------------ | -------------- |
| ![Screenshot from 2022-10-06 21-02-46](https://user-images.githubusercontent.com/83042742/194398362-bcbd2887-3b23-4f5b-bf0b-bc5fefe40208.png) | ![Screenshot from 2022-10-06 21-10-15](https://user-images.githubusercontent.com/83042742/194398498-1bc80039-664b-4146-a8bb-b7aaea2f07e7.png) |

Reply:
I hadn't realized that. But I have implemented them